### DISCUSSION TITLE
How to send HTTP2 Push?

### DISCUSSION THREAD

**Discussion by caramboleyo** [denoland/deno#19984] [Q&A] [answered]
Deno serves HTTP2 and in my tests it works.

I just do not find any code examples or docs about how to send HTTP Pushes.

i guess i have to send it through the buffer stream somehow?

**Comment by lucacasonato** [comment, marked as answer]
HTTP/2 push is deprecated and not supported by browsers anymore. 

Deno provides no support for HTTP/2 push.

**Reply by caramboleyo** [reply]
I am getting old. Things I think are new are already deprecated.
